src/test/java/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterTest.java

```diff
@@ -0,0 +1,116 @@
+package com.atlassian.bitbucket.jenkins.internal.status;
+
+import com.atlassian.bitbucket.jenkins.internal.client.*;
+import com.atlassian.bitbucket.jenkins.internal.client.exception.BitbucketClientException;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketBuildStatus;
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.TaskListener;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.PrintStream;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BuildStatusPosterTest {
+
+    private static final String REVISION_SHA1 = "67d71c2133aab0e070fb8100e3e71220332c5af1";
+    private static final String SERVER_ID = "FakeServerId";
+    private static final String SERVER_URL = "http://www.example.com";
+
+    @ClassRule
+    public static JenkinsRule jenkins = new JenkinsRule();
+
+    @Mock
+    private BitbucketRevisionAction action;
+    @Mock
+    private AbstractBuild build;
+    @InjectMocks
+    private BuildStatusPoster buildStatusPoster;
```

Comment:
move this to the last line of this block, so you group your members by their annotations. I appreciate the alphabetical ordering but I think `ClassRules` first, then `Captors`, then `Mocks` then `Spys` and last the `InjectMock` and each group is alphabetically ordered by type (and then by member name)